config/locales/en.yml

```diff
@@ -709,7 +709,7 @@ en:
       title: "Commission and Payment processing fee"
       stripe: Stripe
       paypal: PayPal
-      info: "Payments in %{service_name} are processed by either Stripe (credit/debit card) or PayPal. %{service_name} will charge a commission depending on the payment gateway. The payment gateway also charges a fee for each payment."
+      info: "Payments in %{service_name} are processed by either Stripe (credit/debit card) or PayPal. %{service_name} will charge a commission depending on the payment gateway."
```

Comment:
@Luis-RG But actually PayPal (not Stripe) will also charge fees. So maybe use something like "Payments in %{service_name} are processed by either Stripe (credit/debit card) or PayPal. %{service_name} will charge a commission depending on the payment gateway and PayPal (but not Stripe) will charge a processing fee."?